doc/docs/master/concepts/pipeline-concepts/pipeline/spout.md

```diff
@@ -83,3 +83,27 @@ The following text is an example of a minimum specification:
   }
 }
 ```
+
+## Resuming Spout Progress
+
+When a spout container crashes, all incomplete operations
+that were processed before the crash are lost, and the spout needs
+to start the interrupted data operation from scratch.
+To keep the history of changes, so that the spout can
+continue where it left off after the restart, you can
+configure a record tracking `marker` file for your spout.
+
+When you specify the `marker` parameter in the
+`transform` section of your pipeline, Pachyderm creates
```

Comment:
maybe mention that it's a subfield of the Spout part of the Input in the pipeline spec, and not really part of the transform